app/frontend/Components/Answer.razor

```diff
@@ -59,7 +59,7 @@
                          ToolTip="Show the supporting content." Disabled="@(Retort is { DataPoints: null } or { DataPoints.Length: 0 })">
                 <ChildContent>
                     <MudPaper Class="pa-2" Elevation="3">
-                        <SupportingContent DataPoints="Retort.DataPoints" />
+                        <SupportingContent DataPoints="Retort.DataPoints  " />
```

Comment:
Is this extra space supposed to be here?

Reply:
@LittleLittleCloud

Reply:
Oh Nope, let me revert it